Makefile

```diff
@@ -258,6 +262,8 @@ kcm-deploy: helm
 
 .PHONY: dev-deploy
 dev-deploy: ## Deploy KCM helm chart to the K8s cluster specified in ~/.kube/config.
+	@mkdir -p $(PROVIDER_TEMPLATES_DIR)/kcm/files/providers
+	@cp -a providers/*.yml $(PROVIDER_TEMPLATES_DIR)/kcm/files/providers/
```

Comment:
Don't we need to do that before the kcm template generation instead? Right now, those changes will be effective on `dev-apply` only but what about the released version of kcm?

Reply:
Can you suggest a correct place to put those commands in? (Make Target)
I had a hard time finding this one.